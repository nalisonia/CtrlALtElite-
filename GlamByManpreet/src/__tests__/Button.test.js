import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../Button';

test('it renders correctly with a label', () => {
  const tree = renderer
    .create(<Button label="Click Me" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
