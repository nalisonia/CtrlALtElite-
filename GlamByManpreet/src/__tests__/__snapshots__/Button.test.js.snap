// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly with a label 1`] = `
<button>
  Click Me
</button>
`;
